index.html

```diff
@@ -69,7 +69,1116 @@
     </nav>
```

Comment:
## Aim to use [`semantic HTML`](https://www.w3schools.com/html/html5_semantic_elements.asp)
- [ ] To achieve this you might want to validate the markup using this [tool](https://validator.w3.org/nu/#textarea) to be sure. Be free to share your thoughts with me about this change, please. 😄